src/foam/u2/ActionView.js

```diff
@@ -63,6 +63,7 @@ foam.CLASS({
   properties: [
     'data',
     'action',
+    [ 'nodeName', 'button'],
```

Comment:
Happened one day in a design meeting.

I think both are equally flexibly, when you extend an element you're probably going to write a custom initE() so you can just set the nodeName there.

I personally like the style of initE() more or less completely defining the element.  It also keeps it more consistent when nesting elements.  If you have a big block of DSL, you get used to configuring elements though DSL, it becomes an impedance mis-match to configure your element through properties but configure all element you create in initE() through DSL.

That was one of my major pain points with foam U1, was all half the time you had to remember which things to do as properties, which as toHTML, which as toInnerHTML.  In U2 its all in initE()

It's why .setNodeName() was added to the DSL.